packages/tooltip/src/Tooltip.js

```diff
@@ -42,6 +43,7 @@ export default function Tooltip(props) {
           container={getTooltipContainer}
           transition
           placement={`${placementY}-${placementX}`}
+          id={tooltipId}
```

Comment:
I don't think adding IDs on their own affects the accessibility so I would revert this. 

The IDs are useful when other things reference them, such as `htmlFor` properties on label elements which reference the IDs of their corresponding input elements.